### PR TITLE
Contact link added to the bottom of the page #134

### PR DESCRIPTION
**Pull request by christofa**
Removed the contact link from the navigation tab and worked on the margin problem as requested.
Screenshot (127)

### CODE REVIEW

File: packages/main/src/pages/static/contact-us/components/ContactFormContainer.jsx [hunk 1]
```diff
@@ -31,14 +31,13 @@ function ContactFormContainer() {
...
 
-  useEffect(() => {
-    getUserInfo().then(userInfo => {
-      setUserAuth(userInfo?.user.email ? userInfo.user : {});
-      setValues(values => ({
-        ...values,
-        email: userAuth.email ? userAuth.email : values.email
-      }));
-    });
+  useEffect(async () => {
```
Comment (JulianaSau) [inline packages/main/src/pages/static/contact-us/components/ContactFormContainer.jsx:34]: Your work was to add contact link to the bottom of the page. How is this code related to the work you were to do?

Comment (christofa) [reply packages/main/src/pages/static/contact-us/components/ContactFormContainer.jsx:34]: Hi, please I am a bit confused as the question isn’t clear. A mentor told me I should make the contact us page below the page and above functional. Please can you shed more light on the question so I can make the corrections. Thanks

Comment (JulianaSau) [reply packages/main/src/pages/static/contact-us/components/ContactFormContainer.jsx:34]: I've confirmed your task. Yours was to remove the link to contact from the navigation bar. It should only be at the bottom of the page @christofa

Comment (christofa) [reply packages/main/src/pages/static/contact-us/components/ContactFormContainer.jsx]: Hi Sau, I have made all the updates you requested

**Review by JulianaSau — APPROVED** [review]

**Comment by JulianaSau** [comment]
Pull changes from the dev branch to update your pr with the recent changes to the repo. Then tag me when you've done so

**Comment by christofa** [comment]
@JulianaSau  All done, please confirm

**Comment by JulianaSau** [comment]
Your branch is showing that its 38 commits behind
Use this command `git pull upstream dev` on the branch you're working on. Then push the changes to your remote repository

**Comment by christofa** [comment]
Thank you so much @JulianaSau. I have done as requested
